File: PooCC01CNetz/Semana05/Java/Ejercicios01/Empleado.java
public class Empleado{
    private String name;
    private double salary;
    private Date fecNac;

    Empleado(String name, double salary, Date fecNac){
        this.name = name;
        this.salary = salary;
        this.fecNac = fecNac;
    }

    Empleado() {}

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setSalary(double salary) {
        this.salary = salary;
    }

    public double getSalary() {
        return salary;
    }

    public void setFecNac(Date fecNac) {
        this.fecNac = fecNac;
    }

    public Date getFecNac() {
        return fecNac;
    }

    public double calcQuincena(){
        return salary * 15;
    }
}

File: PooCC01CNetz/Semana05/Python/Ejercicios01/Empleado.py
from Date import *
class Empleado:
    def __init__(self, name = None, salary = None, fecNac = None):
        if name is None and salary is None and fecNac is None:
            self.__name = None
            self.__salary = None
            self.__fecNac = None
        else:
            self.__name = name
            self.__salary = salary
            self.__fecNac = fecNac
    
    def setName(self, name):
        self.__name = name

    def getName(self):
        return self.__name
    
    def setSalary(self, salary):
        self.__salary = salary

    def getSalary(self):
        return self.__salary
    
    def setFecNac(self, fecNac):
        self.__fecNac = fecNac

    def getFecNac(self):
        return self.__fecNac
    
    def calcularQuincena(self):
        return self.__salary * 15